Gemfile

```diff
@@ -11,9 +11,14 @@ gem 'jwt'
 gem 'bootsnap', '>= 1.1.0', require: false
 gem 'iplocate'
 gem 'fast_jsonapi'
+gem 'rack-cors'
+gem "figaro"
 
 group :development, :test do
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'rspec-collection_matchers', '~> 1.2'
+  gem 'rspec-rails', '~> 3.8'
+  gem 'shoulda', '~> 3.5'
```

Comment:
`shoulda` gem is just a container. It consists of two gems: `shoulda-matchers` and `shoulda-context`. The second one is not applicable here in this project. It's used for `Minitest` and `Test::Unit` test frameworks while you use `Rspec` here. So it can be changed to `gem 'shoulda-matchers'`.